Localizable.strings

```diff
@@ -0,0 +1,9 @@
+/* 
+  Localizable.strings
```

Comment:
Remove the strings file and will properly handle this in a follow up PR. Ticket here: https://github.com/mapzen/ios/issues/187